packages/sync-actions/test/product-sync-prices.spec.js

```diff
@@ -167,4 +167,183 @@ describe('Actions', () => {
     const actions = productsSync.buildActions(now, before)
     expect(actions).toEqual([])
   })
+
+  describe('without `priceID`', () => {
+    let actions
+    const dateNow = new Date()
+    const twoWeeksFromNow = new Date(Date.now() + 12096e5) // two weeks from now
+    const threeWeeksFromNow = new Date(Date.now() + 12096e5 * 1.5)
+
+    const before = {
+      id: '123-abc',
+      masterVariant: {
+        id: 1,
+        prices: [
+          {
+            // change
+            id: '111',
+            value: { currencyCode: 'EUR', centAmount: 3000 },
+            country: 'US',
+            customerGroup: { typeId: 'customer-group', id: 'cg1' },
+            channel: { typeId: 'channel', id: 'ch1' },
+          },
+          {
+            // change
+            id: '333',
+            value: { currencyCode: 'SEK', centAmount: 10000 },
+            country: 'US',
+            channel: { typeId: 'channel', id: 'ch1' },
+          },
+          {
+            // keep
+            id: '444',
+            value: { currencyCode: 'SEK', centAmount: 25000 },
+            country: 'SE',
+          },
+          {
+            // remove
+            id: '666',
+            value: { currencyCode: 'GBP', centAmount: 1000 },
+            country: 'UK',
+            validFrom: twoWeeksFromNow,
+            validUntil: threeWeeksFromNow,
+          },
+          {
+            // change
+            id: '777',
+            value: { currencyCode: 'GBP', centAmount: 1000 },
+            country: 'UK',
+            validFrom: dateNow,
+            validUntil: twoWeeksFromNow,
+          },
+        ],
+      },
+    }
+    const now = {
+      id: '456-def',
+      masterVariant: {
+        id: 1,
+        prices: [
+          {
+            // change
+            value: { currencyCode: 'EUR', centAmount: 4000 },
+            country: 'US',
+            customerGroup: { typeId: 'customer-group', id: 'cg1' },
+            channel: { typeId: 'channel', id: 'ch1' },
+          },
+          {
+            // change
+            value: { currencyCode: 'SEK', centAmount: 15000 },
+            country: 'US',
+            channel: { typeId: 'channel', id: 'ch1' },
+          },
+          {
+            // change
+            value: { currencyCode: 'GBP', centAmount: 10000 },
+            country: 'UK',
+            validFrom: dateNow,
+            validUntil: twoWeeksFromNow,
+          },
+          {
+            // keep
+            value: { currencyCode: 'SEK', centAmount: 25000 },
+            country: 'SE',
+          },
+          {
+            // add
+            value: { currencyCode: 'GBP', centAmount: 1000 },
+            country: 'US',
+            validFrom: twoWeeksFromNow,
+            validUntil: threeWeeksFromNow,
+          },
+        ],
+      },
+    }
+
+    const shufflePrices = array =>
```

Comment:
You could just use https://www.npmjs.com/package/lodash.shuffle

Reply:
Ahh, nice one. Didnt know they had this 👍